services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -706,19 +773,16 @@ public void processDlqRequestForwarded() {
         individualRequest.setRequestStatus(IIndividualRequest.IndividualRequestStatus.FORWARDED.getStatus());
 
         this.mockIndividualRequestDao.persist(individualRequest);
```

Comment:
This line of code was used to set up expected behaviour under EasyMock whilst it was in "record" mode so it can be checked later on. Under Mockito it's purpose is covered by the following verify() line, so I don't think it is needed.

Once it's been removed the following verify() line will need to be moved after the call to processDLQRequest() as that's when the expected behaviour will take place.

Reply:
Line is still present (processDlqRequestForwarded()).

Reply:
done